### DISCUSSION TITLE
How to set menuIcon with svg style?

### DISCUSSION THREAD

**Discussion by huuvu18021438** [ng-alain/ng-alain#2496] [General]
![Screenshot 2024-03-21 143013](https://github.com/ng-alain/ng-alain/assets/72860563/ec3aede3-846b-416e-b9c2-75377d8280d9)
![Uploading Screenshot_1.png…]()

**Comment by huuvu18021438** [comment]
![Screenshot_1](https://github.com/ng-alain/ng-alain/assets/72860563/06199ebb-ea3b-4761-a266-3be476479a1c)

**Reply by cipchk** [reply]
Type `svg` only supports svg content, u can set the type to `img`.

**Reply by huuvu18021438** [reply]
I want to set the menu with my icon without using the library icon. How can I make it change color when clicking or hovering on the menuItem?

**Reply by cipchk** [reply]
Multiple states of `svg` are not supported. If you just want to simply change the color of `.svg`, you can refer to https://blog.union.io/code/2017/08/10/img-svg-fill/
